pkg/server/server.go

```diff
@@ -38,8 +41,9 @@ func setCertificateExpirationDays() error {
 }
 
 const (
-	port      = 8443
-	namespace = "cattle-system"
+	// TODO: Port from env var
+	port      = 9443
+	namespace = "cattle-csp-system"
```

Comment:
Is there something somewhere that is going to create this namespace? I don't see it in the chart.

Reply:
I can create it in the chart if you want, but typically you can do this with `helm install --namespace cattle-csp-system --create-namespace`. FWIW, core rancher appears to take a similar approach, with the [docs](https://rancher.com/docs/rancher/v2.6/en/installation/install-rancher-on-k8s/#install-the-rancher-helm-chart) recommending that you manually create `cattle-system` before you install rancher.

That being said, I definitely see the use case of creating with the chart so if you feel strongly on this let me know.

Reply:
Resolving due to change in goals with this PR.

Reply:
I have left this as is - I think that having the users specify those options on install (or create the namespace beforehand) mirrors what rancher does, and does fit within acceptable conventions. If you disagree, please let me know.